docs(plugins): update plugin listing and add Asana instructions (#1432)
getsentry/sentry-docs: src/collections/_documentation/server/plugins.md

-   Amazon SQS
-   Asana
-   Heroku
-   Pagerduty
-   Phabricator
-   Pivotal
-   Pushover
-   Redmine
-   Segment
-   Sessionstack
-   Splunk
-   Trello
-   Victorops

Most of the official plugins can be found in the [official sentry plugins repository](https://github.com/getsentry/sentry-plugins).

## Deprecated plugins

The following plugins are deprecated and have been replaced with Sentry's built in [Global Integrations]({%- link _documentation/workflow/integrations/index.md -%}).

* Bitbucket
* Clubhouse
* GitHub
* GitLab
* Jira
* Slack
* Visual Studio Team Services

## 3rd Party Plugins {#rd-party-plugins}

> The following plugins are available and were created by members of the Sentry community. Please note that many of these plugins have not been maintained may not work with the latest versions of sentry.

Reviewer: ```suggestion
The following plugins are available and were created by members of the Sentry community. Please note that many of these plugins have not been maintained may not work with the latest versions of Sentry.
```

> The following plugins are available and were created by members of the Sentry community. Please note that many of these plugins have not been maintained may not work with the latest versions of Sentry.

Reviewer: ```suggestion
The following plugins are available and were created by members of the Sentry community. Please note that many of these plugins have not been maintained and may not work with the latest versions of Sentry.
```

-   [sentry-youtrack](https://github.com/bogdal/sentry-youtrack)
-   [sentry-zabbix](https://github.com/m0n5t3r/sentry-zabbix)
-   [sentry-zendesk](https://github.com/ESSS/sentry-zendesk)

## Setup

> Many of these plugins require configuration to work. The instruction on how to configure your environment for these plugins are contianed below.

Reviewer: ```suggestion
Many of these plugins require configuration to work. The instructions on how to configure your environment for these plugins are below.
```

### Asana

You'll have to create an application in Asana to get a client ID and secret. Use the following for the redirect URL:

    <URL_TO_SENTRY>/account/settings/social/associate/complete/asana/

> Ensure you've configured Asana auth in Sentry::

Reviewer: ```suggestion
Ensure you've configured Asana auth in Sentry:
```
